test/DatabaseIntegrationTest/Tests/ProcessDatabase/ProcessDatabaseTests.cpp

```diff
@@ -0,0 +1,79 @@
+#include "stdafx.h"
+//#include "Utils.h"
+//#include "SnowCore/Services/Model/Query/ApplicationVersionQueryService.h"
+//
+//#include "DbAdapterInterface/IDatabase.h"
+//#include "DbSQLiteAdapter/Connection.h"
+//#include "DbSQLiteAdapter/ConnectionConfiguration.h"
+//
+//#include "SecurityConfig/Keys.h"
+//
+//#include <boost/assign/list_of.hpp>
+//
+//using namespace testing;
+//using testing::_;
+//
+//namespace snow {	namespace user {	namespace dbtest { 
+//		
+//		using namespace boost::assign;
+//		
+//		class ProcessDatabaseTests : public testing::Test
+//		{
+//		public:
+//
+//			std::unique_ptr<systelab::db::IDatabase> createDatabase(const std::string& databaseName, const std::string& schemaName, systelab::db::IConnection& connection, std::string sqlFile)
+//			{
+//				if (boost::filesystem::exists(databaseName))
+//				{
+//					boost::filesystem::remove(databaseName);
+//				}
+//				auto key = security::config::keys.find(security::config::USER_DB);
+//				systelab::db::sqlite::ConnectionConfiguration dbConfiguration(databaseName, key->second());
+//
+//				std::unique_ptr<systelab::db::IDatabase> db = connection.loadDatabase(dbConfiguration);
+//				std::string defaultDatabaseStatements;
+//				readFileToString(schemaName, defaultDatabaseStatements);
+//				db->executeMultipleStatements(defaultDatabaseStatements);
+//				defaultDatabaseStatements = "";
```

Comment:
Why is this commented?

Reply:
Because these tests are not completed/available. I would leave them there as a reference for the moment when they are properly implemented.